packages/react-router-dom-v5-compat/rollup.config.js

```diff
@@ -29,26 +30,16 @@ module.exports = function rollup() {
         sourcemap: !PRETTY,
         banner: createBanner("React Router DOM v5 Compat", version),
       },
-      external: [
-        "history",
-        "@remix-run/router",
-        "react",
-        "react-dom",
-        "react-router",
-        "react-router-dom",
-      ],
+      external: (id) => isBareModuleId(id),
```

Comment:
Externalize all `node_modules` deps like we did in Remix, instead of maintaining a manual list